pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -42,67 +42,68 @@ type VolumeZone struct {
 }
 
 var _ framework.FilterPlugin = &VolumeZone{}
+var _ framework.PreFilterPlugin = &VolumeZone{}
 var _ framework.EnqueueExtensions = &VolumeZone{}
 
 const (
 	// Name is the name of the plugin used in the plugin registry and configurations.
 	Name = names.VolumeZone
 
+	stateKey framework.StateKey = Name
```

Comment:
Let's do the same as other plugins.

```suggestion
	preFilterStateKey framework.StateKey = "PreFilter" + Name
```